### PR TITLE
Added missing javadocs, fix formatting.

### PR DESCRIPTION
**Pull request by Articdive**

### CODE REVIEW

File: src/main/java/com/dreamhorizon/enchantments/enchantments/LifeLeech.java [hunk 1]
```diff
@@ -30,27 +30,31 @@
...
-            )
+                new NamespacedKey(DHCore.getPlugin(DHCore.class), "lifeleech"),
+                "Life Leech",
+                1,
+                2,
+                EnchantmentTarget.WEAPON,
+                false,
+                false,
+                new ArrayList<>(),
+                Arrays.asList(
```
Comment (CollinHerber) [inline src/main/java/com/dreamhorizon/enchantments/enchantments/LifeLeech.java:48]: We need to sync on our intellij formatting options, not going to have this battle each time.

File: src/main/java/com/dreamhorizon/enchantments/enchantments/Exhaust.java [hunk 2]
```diff
@@ -32,6 +32,10 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 
+/**
```
Comment (CollinHerber) [inline src/main/java/com/dreamhorizon/enchantments/enchantments/Exhaust.java:35]: Are these required? I don't really care to see author on each of the files. That's what git is for.

File: src/main/java/com/dreamhorizon/enchantments/enchantments/Poison.java [hunk 3]
```diff
@@ -25,34 +25,38 @@
 import org.bukkit.NamespacedKey;
 import org.bukkit.enchantments.EnchantmentTarget;
 import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack;
```
Comment (CollinHerber) [inline src/main/java/com/dreamhorizon/enchantments/enchantments/Poison.java:28]: What was the purpose of this change? Do you like to keep a certain order?

**Review by CollinHerber — COMMENTED** [review]
Just a few comments regarding formatting and requirements

File: src/main/java/com/dreamhorizon/enchantments/enchantments/Poison.java (hunk 3 above)
Comment (Articdive) [reply src/main/java/com/dreamhorizon/enchantments/enchantments/Poison.java:28]: alphabetical with direct classes infront

File: src/main/java/com/dreamhorizon/enchantments/enchantments/Exhaust.java (hunk 2 above)
Comment (Articdive) [reply src/main/java/com/dreamhorizon/enchantments/enchantments/Exhaust.java:35]: No, but then we'll remove them on every file.

File: src/main/java/com/dreamhorizon/enchantments/enchantments/LifeLeech.java (hunk 1 above)
Comment (Articdive) [reply src/main/java/com/dreamhorizon/enchantments/enchantments/LifeLeech.java:48]: Sure, if I can use the auto-format to have it in your syntax it's fine.

**Pull request merged**